src/NUnitFramework/framework/Constraints/PropertyConstraint.cs

```diff
@@ -62,11 +62,7 @@ public override ConstraintResult ApplyTo<TActual>(TActual actual)
                 actualType = actual.GetType();
 
             PropertyInfo property = actualType.GetProperty(name,
-#if PORTABLE
                 BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance);
-#else
-                BindingFlags.Public | BindingFlags.NonPublic | BindingFlags.Instance | BindingFlags.GetProperty);
```

Comment:
BindingFlags.GetProperty is not available in Portable or DNX, but it was also not being used correctly here. GetProperty is used to get the value of the property, not to get the getter property.